kinetic/distribution.yaml

```diff
@@ -9642,6 +9642,11 @@ repositories:
       url: https://github.com/ros-geographic-info/open_street_map.git
       version: master
     status: maintained
+  open_vins:
+    doc:
```

Comment:
could you also add a `source` entry?

Reply:
What is the difference, I saw many packages have the `source` but the utility I was using didn't generate that at all:
https://index.ros.org/contribute/add_repo/

I should add this for all of them?

Reply:
It's not really needed.
I think some tools uses the `source` entry as a reference instead of the `doc` one, but I'm not really sure.
If you only want the package indexed at ros index, adding only a doc entry is fine.

You can also enable pull request testing in the `source` entry: http://wiki.ros.org/buildfarm/Pull%20request%20testing.